src/WPF/ArcGISRuntime.WPF.Samples/Samples/Map/SearchPortalMaps/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Name": "Search a portal for maps",
+  "SampleName": "SearchPortalMaps",
+  "Description": "This sample demonstrates searching a portal for web maps and loading them in the map view. You can search ArcGIS Online public web maps using tag values or browse the web maps in your account.",
+  "Instructions": "",
```

Comment:
Perhaps some additional comments in the 'Instructions' area would be helpful. I was confused by the initial dialog of 'OAuth Settings'. I just clicked the 'Save OAuth Settings' button and I was taken to the next dialog where I was able to search various portal items and add them into my map.